node/tests/webView2Test.ts

```diff
@@ -0,0 +1,41 @@
+import { test as base } from '@playwright/test';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import childProcess from 'child_process';
+
+export const test = base.extend({
+  browser: async ({ playwright }, use, testInfo) => {
+    const cdpPort = 10000 + testInfo.workerIndex;
```

Comment:
It will be helpful if we could create dedicated functions for generating random port numbers and temp folders, so that test users just copy and paste them to their test projects.

Reply:
The webView2 is made for copy/paste. I modified all the examples to make executablePath more visible.